README.md

```
@@ -109,3 +109,45 @@ When invoking `main`, you can pass an `env` object to control behavior:
   if present in `env`, full error details (including `String(err)`) will be appended to stderr
 
 All other `env` properties behave like shell variables available for `${VAR}` expansions.
+## Auto-Generating Tests
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)